docs/index.rst

```diff
@@ -1,105 +1,104 @@
-JAX reference documentation
-===========================
+JAX: High-Performance Array Computing
+=====================================
 
-JAX is Autograd_ and XLA_, brought together for high-performance numerical computing and machine learning research.
-It provides composable transformations of Python+NumPy programs: differentiate, vectorize,
-parallelize, Just-In-Time compile to GPU/TPU, and more.
+JAX is Autograd_ and XLA_, brought together for high-performance numerical computing.
+
+.. grid:: 3
+
+   .. grid-item::
+
+      .. card:: Familiar API
+         :class-card: key-ideas
+         :shadow: None
+
+         JAX provides a familiar NumPy-style API for ease of adoption by researchers and engineers. 
+
+   .. grid-item::
+
+      .. card:: Transformations
+         :class-card: key-ideas
+         :shadow: None
+
+         JAX includes composable function transformations for compilation, batching, automatic differentiation, and parallelization.
+
+   .. grid-item::
+
+      .. card:: Run Anywhere
+         :class-card: key-ideas
+         :shadow: None
+
+         The same code executes on multiple backends, including CPU, GPU, & TPU
 
 .. note::
    JAX 0.4.1 introduces new parallelism APIs, including breaking changes to :func:`jax.experimental.pjit` and a new unified ``jax.Array`` type.
    Please see `Distributed arrays and automatic parallelization <https://jax.readthedocs.io/en/latest/notebooks/Distributed_arrays_and_automatic_parallelization.html>`_ tutorial and the :ref:`jax-array-migration`
    guide for more information.
 
-.. toctree::
-   :maxdepth: 1
-   :caption: Getting Started
 
-   installation
-   notebooks/quickstart
-   notebooks/thinking_in_jax
-   notebooks/Common_Gotchas_in_JAX
+.. grid:: 3
 
-.. toctree::
-   :maxdepth: 1
+    .. grid-item::
 
-   jax-101/index
+      .. card:: :material-regular:`rocket_launch;2em` Getting Started
+         :link: beginner-guide
+         :link-type: ref
+         :class-card: getting-started
 
-.. toctree::
-   :maxdepth: 2
+    .. grid-item::
 
-   debugging/index
+      .. card:: :material-regular:`library_books;2em` User Guides
+         :link: user-guide
```

Comment:
This link needs to be `user-guides` now I think

Reply:
Looks like the change didn't make it into the commit?

Reply:
Youre right my apologies

Reply:
Looks like you updated the `class-card` but not the `link`